lib/checks/label/label-content-name-mismatch-evaluate.js

```diff
@@ -38,7 +38,7 @@ function curateString(str) {
 }
 
 function labelContentNameMismatchEvaluate(node, options, virtualNode) {
-  const { pixelThreshold, occuranceThreshold } = options || {};
+  const { pixelThreshold, occurrenceThreshold } = options || {};
```

Comment:
This will also need to account for the old spelling as it passes the option into `subtreeText`

Reply:
I believe I've fixed that. Since pixelThreshold was being unpacked alone, I opted for just directly assigning it rather than by unpacking. Hope this is ok!